{"name": "test_login_fail[case0]", "status": "passed", "parameters": [{"name": "case", "value": "{'username': 'taozi1', 'passwd': '1234567', 'check': 'Invalid username or password. Signon failed.'}"}], "start": 1666079055595, "stop": 1666079056831, "uuid": "30d08266-3750-401e-a976-9ba6054d0385", "historyId": "c167587ec06a148b12a30980b63b5b1d", "testCaseId": "8c4a898127c5b4ed67fea7eda153ddc9", "fullName": "Test_Cases.test_login.Test_login#test_login_fail", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init')"}, {"name": "parentSuite", "value": "Test_Cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "DESKTOP-41KMUJO"}, {"name": "thread", "value": "13904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_Cases.test_login"}]}